### PR TITLE
Meteor Sats not exclusive to station goal

### PR DESCRIPTION
**Pull request by Sishen1542**
Adds the ability to buy meteor sats independently of station goal so in the case of RNG fuckery during secret the station doesn't get tunguska'd to kingdom come.

edit: Also changes the console crate to be the same price as a crate of sats.

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -886,8 +885,7 @@
 /datum/supply_pack/engineering/shield_sat_control
 	name = "Shield System Control Board"
 	desc = "A control system for the Shield Generator Satellite system."
-	cost = 5000
-	special = TRUE
+	cost = 3000
```
Comment (Poojawa) [inline code/modules/cargo/packs.dm]: Undocumented change.

Comment (Sishen1542) [reply code/modules/cargo/packs.dm]: Forgot to mention that, whoops. Think it's silly that the console is more expensive than a crate of shields when it does nothing for you other than enabling the round end greentext.

Comment (Ghommie) [reply code/modules/cargo/packs.dm]: You could make both cost 4000 if that's what ticks you. Meteor satellites are really easy to set up already, especially compared to other station objectives. 8k can be made by completing one or two of the most profitable yet easy bounties early in.

Comment (Sishen1542) [reply code/modules/cargo/packs.dm]: Works for me, balances out them being always available with it being more expensive in the long run due to shield crate cost increase while keeping the base 8K value dynamic. Committed.

**Review by silicons — APPROVED** [review]

**Pull request merged**